<!-- -->
<launch>

  <arg name="initialize_source" default="Rviz"/>
  <arg name="accel_rate" default="1.0"/>
  <arg name="angle_error" default="0.0"/>
  <arg name="position_error" default="0.0"/>
  <arg name="lidar_height" default="1.0"/>
  <arg name="use_ctrl_cmd" default="false" />
  <node pkg="waypoint_follower" type="wf_simulator" name="wf_simulator" output="screen">
    <param name="initialize_source" value="$(arg initialize_source)"/>
    <param name="accel_rate" value="$(arg accel_rate)"/>
    <param name="angle_error" value="$(arg angle_error)"/>
    <param name="position_error" value="$(arg position_error)"/>
    <param name="lidar_height" value="$(arg lidar_height)"/>
  <param name="use_ctrl_cmd" value="$(arg use_ctrl_cmd)"/>
  </node>

</launch>
